lib/ros_value.h

```diff
@@ -251,24 +239,27 @@ class RosValue {
       object_info_.~object_info_t();
     } else if (type_ == Type::array) {
       array_info_.~array_info_t();
+    } else if (type_ == Type::primitive_array) {
+      primitive_array_info_.~primitive_array_info_t();
     } else {
       primitive_info_.~primitive_info_t();
     }
   }
+  RosValue operator=(const RosValue&) {
+    throw std::runtime_error("Can't call this");
```

Comment:
I feel like we should have a better error message than this

Reply:
I actually want to write this, thanks for catching.